src/actors/client/ActorClient/ActorClient.ts

```diff
@@ -41,7 +41,7 @@ export default class ActorClient {
       }
```

Comment:
Does the gRPC client re-use the same instance internally? If not, can we cache it the same way as HTTP?